Podfile

```diff
@@ -69,7 +69,7 @@ abstract_target 'WordPress_Base' do
   end
 
   target 'WordPressTodayWidget' do
-    pod 'WordPressCom-Stats-iOS/Services', '0.7.7'
+      pod 'WordPressCom-Stats-iOS', :git => "https://github.com/wordpress-mobile/WordPressCom-Stats-iOS.git", :commit => "8a6871af520999c1e731e71ac9697f13cc5e8739"
```

Comment:
Make sure we're version-locking to a tag and not a commit before this is actually merged.